extensions/wikia/DesignSystem/services/DesignSystemGlobalNavigationService.class.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+class DesignSystemGlobalNavigationService extends WikiaService {
+	public function index() {
+		$this->setVal( 'model', $this->getData() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "model" 13 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)